core/src/main/java/com/scalar/db/api/DistributedTransactionManager.java

```diff
@@ -182,9 +183,10 @@ DistributedTransaction start(String txId, Isolation isolation, SerializableStrat
    *
    * @param txId the transaction ID
    * @return {@link DistributedTransaction}
-   * @throws TransactionException if resuming the transaction failed
+   * @throws TransactionNotFoundException if the transaction associated with the specified
+   *     transaction ID is not found
    */
-  DistributedTransaction resume(String txId) throws TransactionException;
+  DistributedTransaction resume(String txId) throws TransactionNotFoundException;
```

Comment:
Throw `TransactionNotFoundException` if the transaction associated with the specified transaction ID is not found.